ProductsSimpleAppFront/app/phones/editor-bar/editor-bar.template.html

```diff
@@ -0,0 +1,3 @@
+<ul class="menu">
```

Comment:
Na to nie zwracaj uwagi. Miałem pomysł na dodanie linku dla "administratora", ale na razie to szczątek ;)